package.json

```diff
@@ -68,10 +75,17 @@
     "mq-polyfill": "1.1.8",
     "prettier": "2.2.1",
     "prop-types": "15.7.2",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
     "react-test-renderer": "16.14.0",
-    "rimraf": "3.0.2"
+    "rimraf": "3.0.2",
+    "rollup": "2.42.4",
+    "rollup-plugin-delete": "2.0.0",
+    "rollup-plugin-typescript2": "0.30.0",
+    "ts-jest": "26.5.4",
+    "tslib": "2.1.0",
+    "typescript": "4.2.3",
+    "yalc": "1.0.0-pre.50"
```

Comment:
This is funny how they never released a proper `1.0.0` 😄

Reply:
🙂